### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by adatzer**

### CODE REVIEW

File: testing/jsm/helpers.js [hunk 1]
```diff
@@ -274,8 +265,8 @@ function inOrder(eventsArray, eventsSpecs) {
 
 function getEventTime(event) {
 
-    // the dvce_created_tstamp
-    return event["event"]["parameters"]["dtm"];
+    // the derived_tstamp
+    return event["event"]["derived_tstamp"];
```
Comment (paulboocock) [inline testing/jsm/helpers.js:269]: Did you intend to switch from the device created timestamp to the derived timestamp here? These are two different timestamps. I think `derived_tstamp` might be a better choice than device timestamp but just pointing it out. `collector_tstamp` might be an even better choice as it is more predictable than `derived_tstamp` but not essential for this PR.

File: testing/jsm/helpers.js [hunk 2]
```diff
@@ -327,23 +307,10 @@ function hasValues(values) {
...
-            } else { // must then have ue_px
-
-                let decod_ue_px = base64decode(ev["event"]["parameters"]["ue_px"]);
-
-                ue_pr = JSON.parse(decod_ue_px);
-
-            }
-
-            let data = ue_pr["data"]["data"];
+            let unstruct_ev = ev["event"]["unstruct_event"];
```
Comment (paulboocock) [inline testing/jsm/helpers.js:312]: This is so much nicer to test now 🎉

File: testing/jsm/helpers.js [hunk 3]
```diff
@@ -519,93 +447,6 @@ function compare(expVal, actVal) {
 }
 
 
-/**
- * Decode a base64-encoded string
- * @param {string} encodedData The base64-encoded string
- * @returns {string} The decoded string
- */
-function base64decode(encodedData) {
```
Comment (paulboocock) [inline testing/jsm/helpers.js:527]: Removing this also makes me very happy 🎉 🎉

**Review by paulboocock — APPROVED** [review]
LGTM

File: testing/jsm/helpers.js (hunk 1 above)
Comment (adatzer) [reply testing/jsm/helpers.js:269]: Yes, this was indeed an intended switch, now that the `derived_tstamp` can be accessed in Micro's response. For switching also took into account:
 - https://snowplowanalytics.com/blog/2015/09/15/improving-snowplows-understanding-of-time/
 - https://discourse.snowplowanalytics.com/t/which-timestamp-is-the-best-to-see-when-an-event-occurred/538

Comment (paulboocock) [reply testing/jsm/helpers.js:269]: Thats cool, `derived_tstamp` is fine by me.

**Pull request merged**